### ISSUE TITLE
Error: Element type is invalid - TabItem

### ISSUE THREAD

**Issue by ckyte31** [PedroBern/react-native-collapsible-tab-view#96]
### Current behavior

Getting the following error while trying to use Tabs.Container on iOS:

Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.

Check the render method of `TabItem`.

### Code sample

```
import {
  createCollapsibleTabs,
} from 'react-native-collapsible-tab-view';
import {useAnimatedRef} from 'react-native-reanimated';

const {useTabsContext, ...Tabs} = createCollapsibleTabs();

// ....

function ProgramCreation({route, navigation}) {
  const containerRef = useAnimatedRef();
  const aboutRef = useAnimatedRef();
  const feedRef = useAnimatedRef();

  const [refMap] = useState({
    A: aboutRef,
    B: feedRef,
  });

  const Header = ({title, height = 200}) => {
    return (
      <View style={[{height}]}>
        <Text style={styles.defaultText}>{title}</Text>
      </View>
    );
  };

  const AboutView = () => {
    return (
       <View/>
    )
  }

  const FeedView = () => {
    return (
       <View/>
    )
  }

  return (
    <Tabs.Container
      containerRef={containerRef}
      HeaderComponent={Header}
      headerHeight={200}
      refMap={refMap}>
        <AboutView />
        <FeedView/>
      </Tabs.Container>
  );
}
```
And my babel.config.js
```
module.exports = {
  presets: ['module:metro-react-native-babel-preset'],
  plugins: ['react-native-reanimated/plugin'],
};

```
### What have you tried

I'm currently using bare react native. No expo or typescript.

I've tried installing this and react-native-reanimated using both npm and yarn...same result. 

I was able to successfully run the example app through expo on my phone but no luck using my own app.

react-native: 0.62.2
react-native-collapsible-tab-view: 3.7.1,
react-native-reanimated: 2.0.0-rc.0 (also tried 2.0.0-rc.2)

**Comment by dougkeen** [comment]
I think this is because `TabItem` internally uses `Pressable`, which is only available since RN 0.63.

**Comment by ckyte31** [comment]
Bit of effort to upgrade my RN version to 0.63.3 but this did the trick. Thanks!!
